ql/src/test/results/clientpositive/llap/fold_case.q.out

```diff
@@ -177,7 +180,7 @@ STAGE PLANS:
                           sort order: 
                           Statistics: Num rows: 1 Data size: 8 Basic stats: COMPLETE Column stats: COMPLETE
                           value expressions: _col0 (type: bigint)
-            Execution mode: vectorized, llap
+            Execution mode: llap
```

Comment:
The vectorization was lost because instead of a real table the `_dummy_table` with empty schema is scanned. It affects only this mapper. Rest of the plan (Reducer) is still vectorized.